src/Collection/RequestCollection.php

```diff
@@ -76,8 +76,7 @@ public function get()
             $this->retrieveResources($firstPage);
         }
 
-        if (
-            $firstPage !== $secondPage
+        if ($firstPage !== $secondPage
```

Comment:
PSR-12 describes that, when splitting statements across multiple lines, the first statement must be on the next line. So we should probably revert this back. https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#51-if-elseif-else

Reply:
Done